proc_scope.py

```diff
@@ -1,32 +0,0 @@
-from pyspecdata import *
```

Comment:
see proc_GDS

Reply:
???

Reply:
never mind -- there's nothing of significance here -- I'm cool with a delete